test/unit/views/IleQueryDucrResponseViewSpec.scala

```diff
@@ -50,8 +55,52 @@ class IleQueryDucrResponseViewSpec extends ViewSpec with Injector {
       view().getElementById("previousMovements") must containMessage("ileQueryResponse.previousMovements")
     }
 
-    "render parent" in {
-      view(parent = Some(parentInfo)).getElementById("parentConsignment") must containMessage("ileQueryResponse.parent")
+    "render default status of entry" in {
+      summaryElement(view(), 1).text must be("")
+    }
+
+    "render empty status of entry" in {
+      summaryElement(view(ducrInfo.copy(entryStatus = Some(status.copy(soe = None)))), 1).text must be("")
```

Comment:
All these tests have been moved out of the ducr/mucr view specs and into tests for the individual components.  Also the some of the tests are no longer valid - each missing code now displays no row, not empty value

Reply:
Copied the tests as they are on GitHub. Should be fine now.